router/nginx.conf

```diff
@@ -44,7 +44,11 @@ http {
     add_header       x-wc-router true;
 
     # These are the locations we know exist solely on v2
-    location ~ ^/assets|/async|/explore|/works|/series|/preview|/articles/W|/exhibitions/W|/events/W {
+    location ~ ^/assets|/async|/explore|/works|/series|/preview|/download/articles/W|/exhibitions/W|/events/W {
```

Comment:
Do you need a pipe between `/download` and `/articles`?

Reply:
Yip - added, thanks.